script/audit_deps.py

```diff
@@ -29,12 +29,19 @@
 
 # Ping security team before adding to ignored_npm_advisories
 ignored_npm_advisories = [
-    1700,   # high RxDoS vector, we don't use MDX in storybook
-    1747,    # moderate RxDoS vector, not fixed in @storybook/addon-essentials
-    1751     # moderate RxDoS vector, not fixed in webpack 4
+    # These are issues in dev depencies, no updates are available yet
+    # To-Do (@jumde) - Remove when https://github.com/brave/brave-browser/issues/18662 is fixed
+    1002401,    # https://github.com/advisories/GHSA-93q8-gq69-wqmw
+    1002422,    # https://github.com/advisories/GHSA-hqhp-5p83-hx96
+    1002475,    # https://github.com/advisories/GHSA-4jqc-8m5r-9rpr
```

Comment:
these 3 look relatively serious:

* https://github.com/advisories/GHSA-4jqc-8m5r-9rpr
* https://github.com/advisories/GHSA-c36v-fmgq-m8hx
* https://github.com/advisories/GHSA-33f9-j839-rf8h

could you see if simply deleting `package-lock.json` and `npm_modules`, followed by `npm install`, fixes some of them?

Reply:
- `npm install` after deleting `package-lock.json` and `npm_modules` does not address these yet
- `npm audit fix` does not fix any of these as well